operators/olm-adding-operators-to-cluster.adoc

```diff
@@ -15,4 +15,9 @@ include::modules/olm-installing-from-operatorhub-using-cli.adoc[leveloffset=+2]
 .Additional resources
 
 * xref:../operators/understanding_olm/olm-understanding-operatorgroups.adoc#olm-operatorgroups-about_olm-understanding-operatorgroups[About OperatorGroups]
+
+include::modules/olm-overriding-proxy-settings.adoc[leveloffset=+1]
```

Comment:
The only spacing like this I've found that's required is a blank line before the `include`, otherwise you get a verbatim `leveloffset` in the text. Works fine with no space after it, and I like to see the "Additional resources" list clearly adjacent to the module it's related to.